test/events/issueTemplateFailEvent.json

```diff
@@ -0,0 +1,22 @@
+{
+    "event": "issues",
+    "payload": {
+      "action": "opened",
+      "issue": {
+        "body": "",
+        "title": "ur thing is broken fix it!!",
+        "user": {
+          "login": "hiimbex"
+        }
+      },
+      "repository": {
+        "name": "testing-things",
+        "owner": {
+          "login": "hiimbex"
+        }
+      },
+      "installation": {
+        "id": 35471
+      }
+    }
+}
```

Comment:
this fixture is identical to `test/events/issueSuccessEvent.json`, but I think ideally eventually the test will look more like an offshoot of a specific issue template

Reply:
yeah oops i know sorry i have yet to change it for issue template purposes